packages/cli/src/ceramic-daemon.ts

```diff
@@ -48,6 +58,53 @@ interface MultiQueries {
   queries: Array<MultiQuery>
 }
 
+const ACCESS_LOG_FMT = 'ip=:remote-addr ts=:date[iso] method=:method path=:url http_version=:http-version req_header:req[header] status=:status content_length=:res[content-length] content_type=":res[content-type]" ref=:referrer user_agent=:user-agent elapsed_ms=:total-time[3]';
```

Comment:
Can we keep this as a constant in the LoggerProvider? Will be nice to easily reuse.

Reply:
I guess what I just merged into CAS should go in here too if you don't mind! https://github.com/ceramicnetwork/ceramic-anchor-service/commit/5a604c7e6bceef4a5c49c7b27e8e981631901bbe

The format and tokens

Reply:
I originally tried to put the entire `makeExpressMiddleware` function in the LoggerProvider, but then I realized that was introducing a dependency on `morgan` into the `common` package.  So I decided to move into the daemon directly as the `cli` package is likely to be the only one that starts an HTTP server.  If/when we have another HTTP server then we can move this somewhere more common then

Reply:
@stbrody Yeah that's fair